debian/copyright

```diff
@@ -0,0 +1,41 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: eos-data-distribution
+
+Files: *
+Copyright:
+ © 2016 Endless Mobile, Inc.
+License: LGPL-3+
+
+Files:
+ dbus-service.py
```

Comment:
In the file? One of you two should update the copyright then (probably better than if I do it, since I’m a third party here)